src/raw.rs

```diff
@@ -292,54 +287,64 @@ impl Future for Connect {
 ///
 /// TiKV uses RocksDB's `ColumnFamily` support. You can learn more about RocksDB's `ColumnFamily`s [on their wiki](https://github.com/facebook/rocksdb/wiki/Column-Families).
 ///
-/// By default in TiKV data is stored in three different `ColumnFamily` values, configurable in the TiKV server's configuration:
+/// TiKV data is stored in three different `ColumnFamily` values, configurable in the TiKV server's configuration:
 ///
 /// * Default: Where real user data is stored. Set by `[rocksdb.defaultcf]`.
 /// * Write: Where MVCC and index related data are stored. Set by `[rocksdb.writecf]`.
 /// * Lock: Where lock information is stored. Set by `[rocksdb.lockcf]`.
 ///
 /// Not providing a call a `ColumnFamily` means it will use the default value of `default`.
 ///
-/// The best (and only) way to create a [`ColumnFamily`](ColumnFamily) is via the `From` implementation:
-///
-/// ```rust
-/// # use tikv_client::raw::ColumnFamily;
-///
-/// let cf = ColumnFamily::from("write");
-/// let cf = ColumnFamily::from(String::from("write"));
-/// let cf = ColumnFamily::from(&String::from("write"));
-/// ```
-///
-/// This is a *wrapper type* that implements `Deref<Target=String>` so it can be used like one transparently.
-///
 /// **But, you should not need to worry about all this:** Many functions which accept a
 /// `ColumnFamily` accept an `Into<ColumnFamily>`, which means all of the above types can be passed
 /// directly to those functions.
-#[derive(Default, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
-pub struct ColumnFamily(String);
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub enum ColumnFamily {
```

Comment:
This is user configurable: https://github.com/tikv/tikv/wiki/TiKV-FAQ#what-are-the-features-of-tikv-block-cache

So I don't think this is appropriate.